templates/on_demand_table_backup.html

```diff
@@ -0,0 +1,90 @@
+{% extends "bbq-menu.html" %}
+{% block main %}
+
+<script type="text/javascript">
+  function OnSubmitForm() {
+    var params = {
+      'onDemand': "True"
+    };
+
+    var url = '/tasks/backups/table/' + document.restoreTableForm.projectId.value + '/'
+        + document.restoreTableForm.datasetId.value + '/' + document.restoreTableForm.tableId.value
+    if (document.restoreTableForm.partitionId.value) {
+      url = url + '/' + document.restoreTableForm.partitionId.value
+    }
+    url = url + '?' + encodeQueryData(params);
+
+    var xhttp = new XMLHttpRequest();
+    xhttp.open("GET", url, true);
+    xhttp.setRequestHeader("request_correlation_id", Date.now())
+    xhttp.timeout = 1000 * 3600;
+
+    xhttp.onload = function () {
+      document.getElementById("response").innerHTML = 'Http response: '+ xhttp.status
+    };
+    xhttp.ontimeout = function (e) {
+      document.getElementById("response").innerHTML = "Request timeout: " + e;
+    };
+
+    xhttp.send();
+    return false;
+  }
+</script>
+<h2>On-demand table backup</h2>
+<p class="lead">
+  BBQ will perform on-demand backup of given source table. It performs backup regardless of result returned from shouldBackup predicate.
```

Comment:
I would describe what is ignored. End user shouldn't need to check what's in predicate.

Reply:
done